client/src/main/java/com/alibaba/nacos/client/utils/IPUtil.java

```diff
@@ -26,31 +26,23 @@
 @SuppressWarnings("PMD.ClassNamingShouldBeCamelRule")
 public class IPUtil {
 
-    public static boolean isIPV4(String addr) {
-        if (null == addr) {
-            return false;
-        }
-        String rexp = "^((25[0-5]|2[0-4]\\d|[01]?\\d\\d?)\\.){3}(25[0-5]|2[0-4]\\d|[01]?\\d\\d?)$";
+    private static final String IPV4_PATTERN = "^((25[0-5]|2[0-4]\\d|[01]?\\d\\d?)\\.){3}(25[0-5]|2[0-4]\\d|[01]?\\d\\d?)$";
```

Comment:
I would suggest make `IPV4_PATTERN` type of `Pattern`, then you don't need to compile it every time

Reply:
make sense